packages/wallets/src/components/WalletsAddMoreCard/WalletsAddMoreCard.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useAvailableWallets } from '@deriv/api';
+import WalletsAddMoreCardBanner from '../WalletsAddMoreCardBanner';
+import WalletsAddMoreCardContent from '../WalletsAddMoreCardContent';
+
+type TWalletsAddMoreCard = NonNullable<ReturnType<typeof useAvailableWallets>['data']>[0];
+
+const WalletsAddMoreCard = ({ currency, is_added, landing_company_name }: TWalletsAddMoreCard) => {
+    return (
+        <div className='wallets-add-more__card' key={currency}>
```

Comment:
@henry-deriv we can have wallets with same currencies for different landing_companies(svg, malta) and this key will be not unique in that case, let's add combination of 
 `currency` and `landding_company` for the key